demo/app/activity.android.ts

```diff
@@ -1,5 +1,5 @@
 import {setActivityCallbacks, AndroidActivityCallbacks} from "ui/frame";
-import { urbanAirshipSettings } from './urbanAirshipSettings';
+// import { urbanAirshipSettings } from './urbanAirshipSettings';
```

Comment:
if we are not using this anymore its best to remove commented code. If its needed for reference then can be part of documentation.